ServerCore/Pages/Teams/Play.cshtml.cs

```diff
@@ -29,7 +29,7 @@ public PlayModel(PuzzleServerContext serverContext, UserManager<IdentityUser> us
 
         public SortOrder? Sort { get; set; }
 
-        private const SortOrder DefaultSort = SortOrder.PuzzleAscending;
+        private const SortOrder DefaultSort = SortOrder.GroupAscending;
```

Comment:
Should this have a secondary sort using OrderInGroup?

Reply:
It does, see lines 68 and 71 of the same file.

Reply:
Ah, github reviews. Looks good.